pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -83,7 +83,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		&flags.Wait,
 		"wait",
 		time.Duration(0),
-		"wait for control plane node to be ready (default 0s)",
+		"the length of time to wait for control plane node to be ready (default 0s). Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't wait.",
```

Comment:
Now that this is two full sentences, it would be nice if it started with capitalization.

Reply:
@stmcginnis maybe at this point send an updated PR, I feel bad for how this went but we also can't merge CLA failures and you're suggesting changes anyhow (and the PR is single line ...)